main.py

```diff
@@ -2,91 +2,81 @@
 import yaml
 import base64
-
-import psycopg2
-import pybreaker
-import sqlalchemy.pool as pool
 import requests
-from flask import Flask, request, make_response
-from flask_restful import Api, Resource
-from flask_swagger_ui import get_swaggerui_blueprint
-from webargs import fields, validate
-from webargs.flaskparser import abort, parser
-from http import HTTPStatus
-
-
-@parser.error_handler
-def handle_request_parsing_error(err, req, schema, *, error_status_code, error_headers):
-    print("inside error handler", err.messages)
-    abort(HTTPStatus.BAD_REQUEST, errors=err.messages)
-
-
-# Initialize flask
-app = Flask(__name__)
-api = Api(app)
-app.url_map.strict_slashes = False
-
-### swagger specific ###
-SWAGGER_URL = '/swagger'
-API_URL = '/static/swagger-ui/doc/swagger.yml'
-SWAGGERUI_BLUEPRINT = get_swaggerui_blueprint(
-    SWAGGER_URL,
-    API_URL,
-    config={
-        'app_name': "ortelius-ms-textfile-crud"
-    }
-)
-app.register_blueprint(SWAGGERUI_BLUEPRINT, url_prefix=SWAGGER_URL)
-### end swagger specific ###
-
-# Initialize database connection
+import psycopg2
+from sqlalchemy import create_engine
+from fastapi import FastAPI, Query, Request, Response, HTTPException, responses, status
+from pydantic import BaseModel, Field
+from typing import List, Optional
+
+# Init Globals
+service_name = 'ortelius-ms-textfile-crud'
+
+# Init FastAPI
+app = FastAPI(
+    title="ortelius-ms-textfile-crud",
+    description="TextFile Crud APIs",
+    version="1.0.0",
+    terms_of_service="http://swagger.io/terms/",
+    contact={
+        "email": "xyz@deployhub.com",
+    },
+    license_info={
+        "name": "Apache 2.0",
+        "url": "https://www.apache.org/licenses/LICENSE-2.0.html",
+    },)
+
+# Init db connection
 db_host = os.getenv("DB_HOST", "localhost")
 db_name = os.getenv("DB_NAME", "postgres")
 db_user = os.getenv("DB_USER", "postgres")
 db_pass = os.getenv("DB_PASS", "postgres")
 db_port = os.getenv("DB_PORT", "5432")
-validateuser_url = os.getenv("VALIDATEUSER_URL", "http://localhost:5000")
-
-# connection pool config
-conn_pool_size = int(os.getenv("POOL_SIZE", "3"))
-conn_pool_max_overflow = int(os.getenv("POOL_MAX_OVERFLOW", "2"))
-conn_pool_timeout = float(os.getenv("POOL_TIMEOUT", "30.0"))
-
-conn_circuit_breaker = pybreaker.CircuitBreaker(
-    fail_max=1,
-    reset_timeout=10,
-)
-
-
-@conn_circuit_breaker
-def create_conn():
-    conn = psycopg2.connect(host=db_host, database=db_name, user=db_user, password=db_pass, port=db_port)
-    return conn
-
-
-# connection pool init
-mypool = pool.QueuePool(create_conn, max_overflow=conn_pool_max_overflow, pool_size=conn_pool_size, timeout=conn_pool_timeout)
-
-# health check endpoint
 
+validateuser_url = os.getenv("VALIDATEUSER_URL", "http://localhost:5000")
 
-class HealthCheck(Resource):
-    def get(self):
-        try:
-            conn = mypool.connect()
+engine = create_engine("postgresql+psycopg2://" + db_user + ":" + db_pass + "@" + db_host + "/" + db_name)
```

Comment:
Please add port as well in the connection string. eg- 
`
create_engine("postgresql+psycopg2://" + db_user + ":" + db_pass + "@" + db_host +":"+ db_port + "/" + db_name)
`